Sources/Apodini/Utilities/Tree.swift

```diff
@@ -18,9 +18,14 @@ extension Tree {
 // MARK: - Node
 
 /// `Node` is a wrapper that enables values to be structured in a tree.
-struct Node<T> {
-    let value: T
-    let children: [Node<T>]
+public struct Node<T> {
+    public let value: T
+    public let children: [Node<T>]
+
+    public init(value: T, children: [Node<T>]) {
+        self.value = value
+        self.children = children
+    }
 }
 
 extension Node {
```

Comment:
```suggestion
public extension Node {
```

You can make the enclosing extension public and all its members will use the same access level by default.